README.md

```diff
@@ -1,27 +1,31 @@
-# AWS Lambda Layer with AWS CLI v1
+# Asset with AWS CLI v1
 <!--BEGIN STABILITY BANNER-->
 
 ---
 
-![cdk-constructs: Stable](https://img.shields.io/badge/cdk--constructs-stable-success.svg?style=for-the-badge)
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
 
 ---
 
+> This library is currently under development. Do not use!
+
 <!--END STABILITY BANNER-->
 
 
-This module exports a single class called `AwsCliLayer` which is a `lambda.Layer` that bundles the AWS CLI v1.
+This module exports a single class called `AwsCliAsset` which is an `s3_assets.Asset` that bundles the AWS CLI v1.
 
-Any Lambda Function that uses this layer must use a Python 3.x runtime.
+Any Lambda Function or Lambda Layer that uses this Asset as its code must use a Python 3.x runtime.
```

Comment:
I don't think it makes sense to use this asset as lambda function code, just lambda layer code, right?

Reply:
Updated the phrasing here, since runtimes are chosen for Functions, and not Layers. Layers can only set a list of compatible runtimes, but they don't actually have a runtime.